src/pages/blog/2024-08-14-semantic-nullability-for-application-developers.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: "Semantic Nullability For Application Developers"
+tags: ["spec"]
+date: 2024-09-12
+byline: Alex Reilly
+---
+
+# Semantic Nullability
+
+> This blog post is directed at application developers using GraphQL. If you are a library author, you should read the more detailed feature spec instead.
```

Comment:
We don't really have a single place to point library authors to right now since we're "between implementations" - let's ask them to get involved?

```suggestion
> This blog post is directed at application developers using GraphQL. If you are a library author, the [nullability working group](https://github.com/graphql/nullability-wg) would love to hear from you - file an issue or join a meeting!
```